src/components/ContentListTable/components/EditContentModal/EditContentModal.tsx

```diff
@@ -282,7 +282,8 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
         </Stack>
       }
     >
-      <TableComposable aria-label='Expandable table'>
+      <TableComposable aria-label='Table for edit modal'>
```

Comment:
Syntax is still wrong here - ouiaId is a property of TableComposable and needs to look like an attribute of the virtual DOM here. (inside the `< >`)

Reply:
ahh, so what I was referring to as all-on-one-line style